### ISSUE TITLE
Addition of links required

### ISSUE THREAD

**Issue by aditi5050** [anitab-org/anitab-org.github.io#365]
![linkpr](https://user-images.githubusercontent.com/102579070/196690057-f42a3eee-0b0c-4062-9dfd-95f34e7d6b86.jpg)
In the getting started section shall I links like the addition of link Zulip and so on @tinniaru3005 shall I look forward for raising a pr for this

**Comment by tinniaru3005** [comment]
Yes, I think that can be done. Avoid making too many changes. You can just add a link to the Zulip chat. 

Also, when you take up an issue try solving it and sending a Pull request before getting another one assigned. This will help you avoid confusion while solving the issues and sending the Pull requests.
